contracts/token/ERC1155/IERC1155.sol

```diff
@@ -5,27 +5,92 @@ pragma solidity ^0.6.2;
 import "../../introspection/IERC165.sol";
 
 /**
-    @title ERC-1155 Multi Token Standard basic interface
-    @dev See https://eips.ethereum.org/EIPS/eip-1155
+ * @dev Required interface of an ERC1155 compliant contract, as defined in the
+ * https://eips.ethereum.org/EIPS/eip-1155[EIP].
  */
 interface IERC1155 is IERC165 {
+    /**
+     * @dev Emitted when `value` tokens of token type `id` are transfered from `from` to `to` by `operator`.
```

Comment:
We're using `value` and `amount` inconsistently.

Reply:
We're consistently inconsistent in all our tokens, which use `amount`. `value` is there because it's part of the event, I believe someone mentioned event argument names are somehow part of their signature?

Reply:
They aren't, only the type.